### PR TITLE
refactor: from console to consola 🎉 

### LINKED ISSUE #2153: Remove console.log and replace it with consola package v0
It's time to step up the game on debugging our application and would be superior to have some classy logger

The suggestion is by @vikiival using https://openbase.com/js/consola which comes from the Nuxtjs team
https://github.com/unjs/consola 

Use it with pluggable reporter FancyReporter https://github.com/unjs/consola/blob/master/src/reporters/fancy.js

The best would be if you manage to tell where in the file is console log located so it would short times to locating issue :)

Next is to integrate some Sentry, RollBar, BugSng Trackjs, Honeybaddger, Raygun or what we love 

### Ref
- https://github.com/kodadot/nft-gallery/issues/834

### PR DESCRIPTION
**Pull request by kkukelka**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

Screenshot 2022-04-07 at 19 10 54

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #2153
- [x] inject consola as plugin and replace all error, warn, log (only in .vue) with it

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)

### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default defineNuxtConfig({
 
   // currently we can only use nitro in development https://github.com/nuxt/framework/issues/886
   bridge: {
-    nitro: process.env.NODE_ENV !== 'production',
+    nitro: false,
```
Comment (roiLeo) [inline nuxt.config.js:28]: What's this?

**Review by roiLeo — COMMENTED** [review]
I've never tested consola but why not

File: nuxt.config.js (hunk 1 above)
Comment (kkukelka) [reply nuxt.config.js]: whoops, played around with new nuxt-edge version. just reintroduced it

**Review by roiLeo — APPROVED** [review]

**Comment by yangwao** [comment]
🫣
image

**Comment by roiLeo** [comment]
pay 50

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 166.75 USD/KSM ~ 0.3 $KSM
🧗 [EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW](https://kusama.subscan.io/account/EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)
🔗 [0x814ea94dffcf5f05baaae97ae3f8644c781cc9099285d568f1b3c30f1c95384c](https://kusama.subscan.io/extrinsic/0x814ea94dffcf5f05baaae97ae3f8644c781cc9099285d568f1b3c30f1c95384c)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**